src/routes/userRoutes.js

```diff
@@ -11,18 +11,18 @@ const routes = {};
 
 routes.resetPasswordEmail = async (req, res) => {
   const email = req.body.email;
-  logger.info("request received to send notification");
-
-  if (!email) {
+  const username = req.body.username;
+  if (!email && !username) {
     return res.status(400).json({ error: { message: "invalid input" } });
   }
-  logger.info("user email is", email);
+  const filter = { where: {} };
```

Comment:
technically, this is called a `query`. You pass in a `query` into sequelize model's `findOne` function

Reply:
Fixed